src/plugins/ceramic-feed/server/content-types/ceramic-post/lifecycles.js

```diff
@@ -17,11 +17,17 @@ const LifecycleHooks = new class {
     this.mediaFields = fields.filter(field => 'media' === get(attributes, `${field}.type`))
   }
 
-  async beforeCreate({ params }) {
+  async afterCreate({ params, result }) {
```

Comment:
that's not a final solution.
actually it's very easy. i just implemented it in 20min

please test it and fix if need (see my next commit)